cleanup/cleanup_test.go

```diff
@@ -0,0 +1,270 @@
+/*
+Copyright 2020 gRPC authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cleanup
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"github.com/grpc/test-infra/config"
+)
+
+var _ = Describe("Test Environment", func() {
+	It("supports creation of load tests", func() {
+		err := k8sClient.Create(context.Background(), newLoadTest())
+		Expect(err).ToNot(HaveOccurred())
+	})
+})
```

Comment:
We don't need to test the environment. I removed it from the controller e2etests I'm working on. Believe me. We'll know if it is broken. All of them should fail. 😆

Reply:
I copied that part, since I am not entirely sure if we should have it or not, let me remove it.